WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -611,21 +612,33 @@ private extension ZendeskUtils {
         return "\(formattedCapacity) \(sizeAbbreviation)"
     }
 
-    static func getLogFile() -> String {
+    static func getEncryptedLogUUID() -> String {
 
-        guard let appDelegate = UIApplication.shared.delegate as? WordPressAppDelegate,
-            let logFileInformation = appDelegate.logger.fileLogger.logFileManager.sortedLogFileInfos.first,
-            let logData = try? Data(contentsOf: URL(fileURLWithPath: logFileInformation.filePath)),
-            var logText = String(data: logData, encoding: .utf8) else {
-                return ""
+        let fileLogger = WPLogger.shared().fileLogger
+        let dataProvider = EventLoggingDataProvider.fromDDFileLogger(fileLogger)
+
+        guard let logFilePath = dataProvider.logFilePath(forErrorLevel: .debug, at: Date()) else {
+            return "Error: No log files found on device"
         }
 
-        // Truncate the log text so it fits in the ticket field.
-        if logText.count > Constants.logFieldCharacterLimit {
```

Comment:
Since `Constants.logFieldCharacterLimit` is no longer being used, it can be removed.

Reply:
Ooh nice catch. Thank you!!